### ISSUE TITLE
Add accessibility support

### ISSUE THREAD

**Issue by edwardkang0925** [TEAMMATES/teammates#10382]
Hi,
We have looked into the WCAG guideline and searched for developer’s tools related to accessibility testing. While doing so, we have found an automated program that converts a website “accessible”. (The service we have found is called Accessibe). We couldn’t use the service yet because we don’t own the Teammates’ live website (the service doesn’t seem to support local development), but we want to ask if you are aware of such tools and what would be an advantage to work on the project manually. 

Besides the automated tool, we are planning to work on screen reader support, virtual keyboard, and color contrast based on WCAG 2.1 standard (please let us know if WCAG 2.0 is what you guys are planning to follow). 

We would appreciate your opinion on our rough starting idea. 
Thank you!

**Comment by madanalogy** [comment]
> Besides the automated tool, we are planning to work on screen reader support, virtual keyboard, and color contrast based on WCAG 2.1 standard (please let us know if WCAG 2.0 is what you guys are planning to follow).

This looks good, you can go ahead and begin development @damithc @xpdavid @wkurniawan07 fyi in case there are any issues with their proposed development plans

**Comment by damithc** [comment]
> Besides the automated tool, we are planning to work on screen reader support, virtual keyboard, and color contrast based on WCAG 2.1 standard (please let us know if WCAG 2.0 is what you guys are planning to follow).

@edwardkang0925 Sounds good. In the meantime, you can keep a lookout for automated tools that are less restrictive and can be used without having owner access to the live site. As for which standard to follow, you can also research and make a recommendation yourself (with pros and cons of each option).

**Comment by wkurniawan07** [comment]
Closing due to lack of scope, planning, and progress.
